cmd/routed-eni-cni-plugin/driver/driver.go

```diff
@@ -216,6 +225,48 @@ func (createVethContext *createVethPairContext) run(hostNS ns.NetNS) error {
 	return nil
 }
 
+// Implements `SettleAddresses` functionality of the `ip` package.
+// WaitForAddressesToBeStable waits for all addresses on a link to leave tentative state.
+// Will be particularly useful for ipv6, where all addresses need to do DAD.
+// If any addresses are still tentative after timeout seconds, then error.
+func WaitForAddressesToBeStable(ifName string, timeout int) error {
+	link, err := netlink.LinkByName(ifName)
+	if err != nil {
+		return fmt.Errorf("failed to retrieve link: %v", err)
+	}
+
+	deadline := time.Now().Add(time.Duration(timeout) * time.Second)
+	for {
+		addrs, err := netlink.AddrList(link, netlink.FAMILY_ALL)
```

Comment:
I think we only want this for v6, so we may as well use that to our advantage here.
```suggestion
		addrs, err := netlink.AddrList(link, netlink.FAMILY_V6)
```